server/tests/integration_test.rs

```diff
@@ -639,6 +620,11 @@ fn reconstruct_record_batch(original: &RecordBatch, query: &RecordBatch) -> Reco
 
 /// Terminates the Arrow Flight Server, and only returns when the process has been terminated.
 fn terminate_arrow_flight_server(mut flight_server: Child) {
-    flight_server.kill().expect("Could not send kill signal");
-    flight_server.wait().expect("Could not exit.");
+    let mut s = System::new_all();
+
+    while let Some(_process) = s.process(Pid::from_u32(flight_server.id())) {
+        flight_server.kill().expect("Could not kill _process.");
```

Comment:
Why use "_process" instead of just "process" or alternatively more detailed information about the process (a PID?)?

Reply:
This is what the Rust compiler tells me to do if I do not use the variable in a `while let` loop. 
![image](https://user-images.githubusercontent.com/43062786/204497316-71836965-604a-4245-9eff-43d06136bf7c.png)

Reply:
Now, the error message is "Could not kill _process" (with an underscore as in the variable name), but I think the error message either should be "Could not kill process" (without the underscore) or something like "Could not kill process 4205"

Reply:
Ahhh, sorry! I completely missed that you were referring to the print and not the code. How do you like these changes?

Reply:
Thanks, I think that is better.